### PR TITLE
WDRT parity - continued

### PR DESCRIPTION
**Pull request by hivanov-nrel**
code:
- [x] add copula methods to environmental contours

examples:
- [x] update environmental contours example to show different copula methods

### CODE REVIEW

**Comment by hivanov-nrel** [comment]
I wont be able to get to the remaining items in Issue #146 and some of them require compatibility with higher version of MHKiT-Python anyways so we'll have to leave that one open.

Also, see my comment on Issue #109 and why I think that one should be closed as well. 

Otherwise this should be ready for review.

File: mhkit/wave/resource/environmental_contours.m [hunk 1]
```diff
@@ -68,9 +89,16 @@
...
+    options.min_bin_count = 40;
+    options.initial_bin_max_val = 1;
+    options.bin_val_size = 0.25;
     options.nb_steps = 1000;
+    options.bandwidth = py.None;
+    options.Ndata_bivariate_KDE = 100;
+    options.max_x1 = py.None;
+    options.max_x2 = py.None;
+    options.PCA = py.None;
+    options.PCA_bin_size = 250;
```
Comment (simmsa) [inline mhkit/wave/resource/environmental_contours.m:101]: I think this is a valid strategy to set these values but we will need to ensure that these don't diverge from the default values set in MHKiT-Python

**Review by simmsa — COMMENTED** [review]
@hivanov-nrel this looks good to me, nice work!

I added one note about setting default values in MATLAB vs Python. I'm not sure what is a good long term strategy for this type of default setting, but it would be ideal to use the default values in MHKiT-Python. In this situation I don't think there is much of a choice without refactoring a significant portion of the MHKiT-Python environmental contours code. Your solution here works and keeps us moving forward.

**Comment by simmsa** [comment]
@hivanov-nrel, I will fix the merge conflict shortly and then I think we can merge this safely...

**Pull request merged**